webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/EditableValueHelper.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  TextInput,
+  TextArea,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+} from '@patternfly/react-core';
+
+const withFormGroup = Component => componentProps => {
```

Comment:
This seems we are implementing a Form helper in a plugin, I'd really love if we would manage to do this in core. There are about three in core already and this is yet another implementation :((

Reply:
I would also love if this was something I did not have to do in a plugin, but core does not provide any PF4 form helpers as far as I looked.